spec/transaction_repo_spec.rb

```diff
@@ -58,9 +58,9 @@
 
       expect(expected.class).to eq(Transaction)
       expect(expected.invoice_id).to eq(8)
-      expect(expected.credit_card_number).to eq(4242424242424242)
-      expect(expected.credit_card_expiration_date).to eq(220)
-      expect(expected.result).to eq("failed")
+      expect(expected.credit_card_number).to eq("4242424242424242")
+      expect(expected.credit_card_expiration_date).to eq("0220")
+      expect(expected.result).to eq(:failed)
     end
 
     it "can update transaction" do
```

Comment:
great refactor overall. we need to remember to remove whitespace when we refactor.